bokehjs/src/coffee/models/ranges/data_range1d.coffee

```diff
@@ -17,6 +17,8 @@ class DataRange1d extends DataRange.Model
       follow_interval: [ p.Number        ]
       default_span:    [ p.Number, 2     ]
       bounds:          [ p.Any           ] # TODO (bev)
+      min_interval: [ p.Any ]
+      max_interval: [ p.Any ]
```

Comment:
`min_interval` and `max_interval` can be anything that `min` and `max` can be, except that they can also be zero. What type should I use here?